src/DotLiquid.Tests/Tags/RawTests.cs

```diff
@@ -20,6 +20,15 @@ public void TestOutputInRaw ()
                 "{% raw %}{{ test }}{% endraw %}");
         }
 
+        [Test]
+        public void TestRawWithErbLikeTrimmingWhitespace()
+        {
+            Helper.AssertTemplateResult("{{ test }}", "{%- raw %}{{ test }}{%- endraw %}");

```

Comment:
If I understand this correctly DotLiquid completely ignores whitespace control from Shopify's spec. So this is just a temporary stop gap while the real work is implementing the whitespace control feature/